inc/utilities/UserEvents.utl.php

```diff
@@ -716,7 +447,6 @@ public function doDelete(ParameterHelper $params) : ReturnHelper {
 			(new UserRoles($this->db, $this->log))->deleteAllForUser($user->id);
 			(new UserRelations($this->db, $this->log))->deleteAllForUser($user->id);
 			(new UserProfiles($this->db, $this->log))->deleteAllForUser($user->id);
-			(new UserDevices($this->db, $this->log))->deleteAllForUser($user->id);
```

Comment:
Why remove the removal of devices?  This change doesn't seem relevant to the addition of the _PRE events.

Reply:
I just found this in here as a leftover of another effort I'd done to remove the Devices stuff.